### DISCUSSION TITLE
Потрібна допомога з перекладом

### DISCUSSION THREAD

**Discussion by levonet** [levonet/arduino-reference-uk#1] [Q&A]
Якщо ти бажаєш допомогти, напиши про це тут.

Як можна допомогти:
- перекладом з англійської на українську
- вичиткою в же перекладеного матеріалу та виправленням помилок
- консультацією з перекладу окремих фраз чи термінів (в телеграм групі)

**Comment by BatsIhor** [comment]
Привіт, приєднуюсь

**Reply by BatsIhor** [reply]
Між іншим як знайти цю телеграм групу?

**Reply by levonet** [reply]
Телеграмгрупи поки немає, можеш створити і додати мене туди )
Можна напряму.
Мій телеграм @levonet
